plugins/jobtype/src/azkaban/jobtype/HadoopJavaJob.java

```diff
@@ -182,6 +183,9 @@ public void run() throws Exception {
               .getHadoopTokens(hadoopSecurityManager, props, getLog());
       getJobProps().put("env." + HADOOP_TOKEN_FILE_LOCATION,
           tokenFile.getAbsolutePath());
+
+      hadoopSecurityManager.updateJobProps(getJobProps(), getSysProps(),
```

Comment:
to provide user a convenient way to resolve the name mapping,  this is no longer needed as we reverted back to the non-mapping route .